.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
and editorconfig is picked up by Atom automatically? Also, windows line endings are stupid. I'll convert the whole repo now.